README.md

```diff
@@ -1,5 +1,7 @@
 # Create Probot Plugin
 
+[![Greenkeeper badge](https://badges.greenkeeper.io/probot/create-probot-plugin.svg)](https://greenkeeper.io/)
```

Comment:
the repo was renamed, can you change it to https://badges.greenkeeper.io/probot/create-probot-app.svg?